client/signup/steps/plans/index.jsx

```diff
@@ -30,12 +30,19 @@ export class PlansStep extends Component {
 	state = {
 		isDesktop: isDesktop(),
 		experiment: null,
+		experimentLoaded: false,
 	};
 
 	componentWillMount() {
-		loadExperimentAssignment( 'disabled_monthly_personal_premium_v2' ).then( ( experimentName ) => {
-			this.setState( { experiment: experimentName } );
-		} );
+		if ( this.props.flowName === 'onboarding' || this.props.flowName === 'launch-site' ) {
+			loadExperimentAssignment( 'calypso_signup_monthly_plans_default_202201_v1' ).then(
```

Comment:
[`ProvideExperimentData`](https://github.com/Automattic/wp-calypso/blob/trunk/packages/explat-client-react-helpers/README.md#api-provideexperimentdata) with the `isEligible` option set might make this simpler.

Reply:
I'd prefer to keep the current way of checking assignment, unless this could lead to some other issue in which case I'm happy to refactor to use the `ProvideExperimentData` component.